blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -20,8 +22,15 @@ const { string, object, any, func } = PropTypes;
 
 export class RunDetailsPipeline extends Component {
     componentWillMount() {
-        if (this.props.fetchNodes) {
-            this.props.fetchNodes(this.generateConfig(this.props));
+        const { fetchNodes, fetchLog, result } = this.props;
+        const mergedConfig = this.generateConfig(this.props);
+
+        if (result && result._class === 'io.jenkins.blueocean.service.embedded.rest.PipelineRunImpl') {
```

Comment:
Sounds like we should put a ticket in the backlog todo just that? Use the
Technical-debt label
On Wed, Jun 22, 2016 at 5:28 PM, Thorsten Scherler <notifications@github.com>
wrote:

> In blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx
> <https://github.com/jenkinsci/blueocean-plugin/pull/285#discussion_r68005764>
> :
>
> > @@ -20,8 +22,15 @@ const { string, object, any, func } = PropTypes;
> >
> >  export class RunDetailsPipeline extends Component {
> >      componentWillMount() {
> > -        if (this.props.fetchNodes) {
> > -            this.props.fetchNodes(this.generateConfig(this.props));
> > +        const { fetchNodes, fetchLog, result } = this.props;
> > +        const mergedConfig = this.generateConfig(this.props);
> > +
> > +        if (result && result._class === 'io.jenkins.blueocean.service.embedded.rest.PipelineRunImpl') {
>
> I agree that this will need to be refactored as soon I read up on the
> capabilities api, @vivek <https://github.com/vivek> where talking about
> it in Japan.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jenkinsci/blueocean-plugin/pull/285/files/e075601e1ca8d87e11997614283e3bd6237356ea#r68005764>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AADD7Iz5brz8PDV6hWFwV7x130Jr2VXsks5qOOQFgaJpZM4I7JLz>
> .
>
-- 
James Dumay
Product Manager
CloudBees, Inc.
Skype: jdumaycb Twitter: i386